frontend/src/modules/scribble/components/RoundNumber.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+    <div class="round-number">
+        <b-container fluid>
+            <b-row align-v="center" class="justify-content-md-center">
+                <b-col md="auto">Round: </b-col>
+                <b-col md="auto"
+                    ><input
+                        class="round-number__input"
+                        type="text"
+                        v-model="roundNum"
```

Comment:
`v-model` is more for two-way bindings, but in this case it's `readonly` and a little misleading. It'd be better to just use a `<p>` tag and reference the value so a reader knows it's not supposed to be input.